src/pages/docs/features/notifications.md

```diff
@@ -2,5 +2,31 @@
 title: Notifications
 description: Using the Cirro notification system.
 ---
+## What are Cirro notifications?
+Communicating with your users is essential in order to maintain a well-functioning web application, thus a crowd-working service. In addition to the Cirro provided tools to make identity management, gig management, and invitation management simpler for you and your workers, we equip you with multiple means of communication.
 
-TODO
+## Terminology
+### Notifications
+Before we clarify technical details, we need to understand notification-related terminology using our tree-planting example.
+All communication with users is managed via **Notifications**. A notification represents a particular message sent to a particular user at a particular point in time. The most common type of notification that a user will receive is the **"gig invitation"** notification that they receive each time when they match the criteria of a gig and have been invited to join using a variety of delivery mechanisms configurable based on project needs, utilising the Cirro worker interface and the user's email address. E.g.: A volunteer ('User' of the 'Space', once approved 'Worker') is invited to participate in a tree-planting event ('Gig') via an email or Cirro interface.
```

Comment:
```suggestion
this is a suggestion
```

Reply:
I'm confused, @thebluber you propose to replace the paragraph with `this is a suggestion`? ^^